package.json

```diff
@@ -97,45 +97,60 @@
         ]
     },
     "devDependencies": {
-        "@angular-devkit/build-angular": "15.2.10",
-        "@angular-devkit/core": "15.2.10",
-        "@angular-devkit/schematics": "15.2.10",
-        "@angular/animations": "15.2.10",
-        "@angular/cdk": "15.2.9",
-        "@angular/cli": "15.2.10",
-        "@angular/common": "15.2.10",
-        "@angular/compiler-cli": "15.2.10",
-        "@angular/core": "15.2.10",
-        "@angular/forms": "15.2.10",
-        "@angular/platform-browser": "15.2.10",
-        "@angular/router": "15.2.10",
+        "@angular-builders/custom-webpack": "16.0.1",
```

Comment:
https://github.com/taiga-family/taiga-ui/blob/8915881b55d50c0da9894fe9b0e3c43ee016ccd1/projects/demo/package.json#L23

Reply:
Remove all extra deps.

And run 
```shell
rm -rf package-lock.json node_modules **/node_modules
npm i --workspaces --include-workspace-root
```